.github/scripts/parse_test_report.py

```
@@ -0,0 +1,198 @@
+        lines.append("</details>")
+
+    lines.append(f"\n[View full run]({run_url})")
+    return "\n".join(lines)
+
+
+def post_to_pr(summary: str, repo: str, pr_number: int, token: str):
+    """Post or update a comment on a GitHub PR using the gh CLI."""
+    marker = "<!-- cross-repo-integration-test -->"
+    body = f"{marker}\n{summary}"
+
+    result = subprocess.run(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)